core/modules/css/python/testKvInterface.py

```diff
@@ -55,56 +110,51 @@ def testCreateGetSetDelete(self):
+        self.assertRaises(lsst.qserv.css.KeyExistsError, self._kvI.create, k1, v1)
+
         # set the value to something else
         self._kvI.set(k1, v2)
         # get it
         v2a = self._kvI.get(k1)
         assert(v2a == v2)
         # delete it
-        self._kvI.delete(k1)
+        self._kvI.deleteKey(k1)
         # try deleting it again, this should fail
-        self.assertRaises(KvException, self._kvI.delete, k1)
+        self.assertRaises(lsst.qserv.css.NoSuchKey, self._kvI.deleteKey, k1)
         # try to get it, it should fail
-        self.assertRaises(KvException, self._kvI.get, k1)
-        # try to set it, it should fail
-        self.assertRaises(KvException, self._kvI.set, k1, v1)
+        self.assertRaises(lsst.qserv.css.NoSuchKey, self._kvI.get, k1)
+        # set it
+        self._kvI.set(k1, v1)
         # get the second key
         v2a = self._kvI.get(k2)
         assert(v2a == v2)
         # test getChildren
-        self._kvI.getChildren("/unittest/")
-        self.assertRaises(KvException, self._kvI.getChildren, "/whatever")
-        # try to set for invalid key
-        self.assertRaises(KvException, self._kvI.set, "/whatever", "value")
+        # try an invalid key first
+        self.assertRaises(lsst.qserv.css.CssError, self._kvI.getChildren, "/unittest/")
+        self.assertEqual(self._kvI.getChildren("/unittest"), ("my", ))
+        self.assertRaises(lsst.qserv.css.NoSuchKey, self._kvI.getChildren, "/whatever")
         # try to delete invalid key
-        self.assertRaises(KvException, self._kvI.delete, "/whatever")
-        # print everything
-        self._kvI.dumpAll()
-
-    #def testPerformance(self):
-    #    n = 10 # set it to something larger for real test...
-    #    start = time.clock()
-    #    for i in range(1,n+1):
-    #        k = "unittest/node_%02i" % i
-    #        # print ("creating %s --> %i" % (k, i))
-    #        self._kvI.create(k, str(i))
-    #    elapsed = time.clock()-start
-    #    print "It took", elapsed, "to create", n, "entries"
-
-
-####################################################################################
-def setLogging():
-    logging.basicConfig(
-        #filename="/tmp/testKvInterface.log",
-        format='%(asctime)s %(name)s %(levelname)s: %(message)s',
-        datefmt='%m/%d/%Y %I:%M:%S',
-        level=logging.DEBUG)
-    logging.getLogger("kazoo.client").setLevel(logging.ERROR)
+        self.assertRaises(lsst.qserv.css.NoSuchKey, self._kvI.deleteKey, "/whatever")
+        # # print everything
+        # self._kvI.dumpAll()
+
+    def testPerformance(self):
+        n = 10 # set it to something larger for real test...
+        start = time.clock()
+        for i in range(1,n+1):
+            k = "/unittest/node_%02i" % i
+            #print ("creating %s --> %i" % (k, i))
+            self._kvI.create(k, str(i))
+        elapsed = time.clock()-start
+        print("It took" + str(elapsed) + "to create" + str(n) + "entries")
```

Comment:
I'm asking for that line to be added to the top of the file so it's clear we are using the print function rather than relying on python2.7 to ignore the parentheses. It will be less confusing later on if someone edits the print by adding a "," to the end and then a tuple prints out.

Adding the line also prevents `futurize` from grumbling.